osh.spec

```diff
@@ -156,23 +156,32 @@ OpenScanHub worker devel configurations.
 
 %build
 
-# Add -s to the shebang in osh/client/osh-cli:
-# https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_shebang_macros
-# TODO: Remove this when we migrate to newer Python packaging macros which
-# do this automatically.
-%py3_shebang_fix osh/client/osh-cli
-
 # collect static files from Django itself
 PYTHONPATH=. osh/hub/manage.py collectstatic --noinput
 
 # set path to python sitelib in the example httpd config
 sed 's|@PYTHON3_SITELIB@|%{python3_sitelib}|' osh/hub/osh-hub-httpd.conf.in > osh/hub/osh-hub-httpd.conf
 
+%if 0%{?fedora} > 41
+%pyproject_wheel
+%else
+# Add -s to the shebang in osh/client/osh-cli:
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_shebang_macros
+%py3_shebang_fix osh/client/osh-cli
+
 %py3_build
+%endif
 
 
 %install
+%if 0%{?fedora} > 41
+%pyproject_install
+mv %{buildroot}%{python3_sitelib}/etc/ %{buildroot}/
+mv %{buildroot}%{python3_sitelib}/usr/{bin,share}/ %{buildroot}/usr/
+mv %{buildroot}%{python3_sitelib}/usr/lib/systemd/ %{buildroot}/usr/lib/
```

Comment:
This should not be necessary, see https://src.fedoraproject.org/rpms/pungi/pull-request/11#comment-268052

Instead, drop the `/usr` prefix from https://github.com/openscanhub/openscanhub/blob/8ba947af7fea949a453049d1dc720422aad872ed/setup.py#L21 https://github.com/openscanhub/openscanhub/blob/8ba947af7fea949a453049d1dc720422aad872ed/setup.py#L28 https://github.com/openscanhub/openscanhub/blob/8ba947af7fea949a453049d1dc720422aad872ed/setup.py#L31 https://github.com/openscanhub/openscanhub/blob/8ba947af7fea949a453049d1dc720422aad872ed/setup.py#L34

Reply:
When I drop the `/usr` prefix, the `osh-cli` binary is installed into `/usr/lib/python3.13/site-packages/bin/osh-cli` instead of `/usr/lib/python3.13/site-packages/usr/bin/osh-cli`, which is not an improvement really.

Reply:
Sorry for the confusion, yes, I meant the `/usr/` prefix -- a.k.a. use paths relative to `/usr`.

This is not possible with `/etc` which is known. Wheels are packaged and than unpacked relative to prefix, and they cannot "leak" outside of it. Installing files to `/etc` from Python wheels is never going to work and we are expected to do this sort of thing on RPM level (either by moving the files or installing them manually or whatever).